code/modules/jobs/job_types/paramedic.dm

```diff
@@ -0,0 +1,45 @@
+/datum/job/paramedic
+	title = "Paramedic"
+	flag = PARAMEDIC
+	department_head = list("Chief Medical Officer")
+	department_flag = MEDSCI
+	faction = "Station"
+	total_positions = 3
+	spawn_positions = 2
+	supervisors = "the chief medical officer"
+	selection_color = "#74b5e0"
+
+	outfit = /datum/outfit/job/paramedic
+
+	access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_GENETICS, ACCESS_CLONING, ACCESS_MINERAL_STOREROOM, ACCESS_MAINT_TUNNELS, ACCESS_EVA, ACCESS_ENGINE, ACCESS_CARGO, ACCESS_HYDROPONICS, ACCESS_RESEARCH)
```

Comment:
Cargo/Hydro/Engi/Sci are not needed
EVA is reasonable but maybe medical like all departments should get a hardsuit or two?

Reply:
EVA isn't reasonable, the only people with default access to EVA are the captain and two heads of staff. Just give them their own softsuit (they're doctors, they don't need a hardsuit) if you really want them to be EVA-ready at shift start.